### PR TITLE
add backup, restore commands

### PR DESCRIPTION
**Pull request by chrismerck**

### CODE REVIEW

File: bond/commands/backup.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+        if args.latest:
+            if len(file_list) == 0:
+                print("No backups found")
+                return
+            args.file = file_list[-1]["file"]
+
+        start_daemon(4444)
+        bondid = BondDatabase.get_assert_selected_bondid()
+        # timestamp = str(int(time.time()))
```
Comment (marciogranzotto) [inline bond/commands/backup.py:155]: are we going to use this timestamp? I commented it out since the linter was complaining about an unused variable

**Review by marciogranzotto — DISMISSED** [review]
I've merged trunk into this branch, auto formatted it and fixed the linter issues

**Comment by marciogranzotto** [comment]
@chrismerck actually can we change the request from `PUT` to `POST`? Also if possible add the content-type multipart. That way we need minimal changes in the server libs (for what I'm researching)

**Comment by chrismerck** [comment]
> Can we change the request from PUT to POST?

It's easy to change. But, problem is, it's definitely a PUT that we want to do. Authoritative reference: https://tools.ietf.org/html/rfc2616#section-9.6  Nice visual comparison here: https://restfulapi.net/rest-put-vs-post/

> Also if possible add the content-type multipart.

I've added `Content-Type: application/octet-stream` in latest commit.

**Review by marciogranzotto — APPROVED** [review]
@chrismerck can this be merged?

**Comment by chrismerck** [comment]
Ok. If it doesn't work, we can fix it :)

**Pull request merged**